src/components/eRACommons.js

```diff
@@ -44,7 +44,7 @@ export const eRACommons = hh(class eRACommons extends React.Component {
     }
     if (isFcUser) {
       const parsedToken = qs.parse(searchArg);
-      this.verifyToken(parsedToken).then(
+      await this.verifyToken(parsedToken).then(
```

Comment:
This is a curious pattern. I'd love to discuss it with you offline.

Reply:
Don't think I need to await here, but we do need to verify